build/build.go

```diff
@@ -84,12 +84,13 @@ type Options struct {
 	NoCacheFilter []string
 	Platforms     []specs.Platform
 	Pull          bool
-	Session       []session.Attachable
 	ShmSize       opts.MemBytes
 	Tags          []string
 	Target        string
 	Ulimits       *opts.UlimitOpt
 
+	Session []session.Attachable
+
```

Comment:
nit: Why do we need to change the order?

Reply:
There's no real reason for this - except to try and match up with the [`controllerapi.BuildOptions` struct](https://github.com/docker/buildx/pull/1614/files/aead0c66e5e070d3a1882ff31860b2088bca18a8#diff-0cae9bac155d49c8975ab4e5c60d7bb85d98dc5226def18dc0daaeb66497e7e5R30-R49), that I'd like to eventually merge into the controller API. 

In conversation with @tonistiigi, we were talking about the possibility of having all buildx builds go through the controller interface, including with bake, etc, so as to reduce the number of possible code paths, though I think we need to keep playing around to try and see if that's possible.